Tests/js/views/ez-searchlistview.html

```diff
@@ -60,13 +60,17 @@
                 fullpath: "../../../../Resources/public/js/extensions/ez-loadmorepagination.js"
             },
             "ez-subitemlistitemview": {
-                requires: ['ez-templatebasedview', 'ez-translateproperty'],
+                requires: ['ez-templatebasedview', 'ez-translateproperty', 'ez-draftconflict'],
```

Comment:
It won't load the "ez-subitemlistitemview"  without the dependency on the extension (added in this PR) and the test will fail.